### PR TITLE
Document the findPlugin Method

### PR DESCRIPTION
**Pull request by pjlamb12**
docs: document the findPlugin function

document the findPlugin and how to use it

ISSUES CLOSED: #1180

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/scullyio/scully/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Other... Please describe:

## What is the current behavior?
There was no documentation for `findPlugin`

Issue Number: #1180

## What is the new behavior?
This provides documentation for `findPlugin`.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: docs/Reference/scully-api/findPlugin.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+title: findPlugin
+published: true
+lang: en
+position: 100
+---
+
+# findPlugin
+
+The [`findPlugin`](https://github.com/scullyio/scully/blob/main/libs/scully/src/lib/pluginManagement/pluginConfig.ts#L49) function retrieves a plugin from Scully and returns it. It takes a single parameter:
```
Comment (SanderElias) [inline docs/Reference/scully-api/findPlugin.md:10]: It does actually accept 3 parameters! 
So, it takes 1 to 3 params. If the first one is a type, the second is the name (name can be a string, or a symbol). the 3rth one is not really for external use, but you can use that if you want to use optional plugins (you need to set it to false then, you get undefined if the plugin isn't there)
BTW, you can have plugins with the same name for different types.

**Review by SanderElias — COMMENTED** [review]
Can you address the missing parameters in the doc?

**Comment by pjlamb12** [comment]
Can you clarify what you mean by that?

**Comment by pjlamb12** [comment]
I just saw another comment and remembered your gitter message, so I will update this.

**Comment by pjlamb12** [comment]
@SanderElias I just updated the documentation with the requested changes.

**Review by SanderElias — APPROVED** [review]

**Pull request merged**